.github/workflows/webext-lint.yml

```
@@ -0,0 +1,22 @@
+name: Web-Ext lint
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: "web-ext lint"
+        uses: kewisch/action-web-ext@v1
```

He has not released a new version [since 2019](https://github.com/kewisch/action-web-ext/releases)...
```suggestion
        uses: kewisch/action-web-ext@master
```

---

https://github.com/kewisch/action-web-ext/issues/27